src/dialog/overflow-menu/overflow-menu.component.ts

```diff
@@ -77,9 +75,16 @@ export class OverflowMenu {
 
 	@Input() placement: "bottom" | "top" = "bottom";
 
-	@ContentChild(OverflowMenuDirective) overflowMenuDirective: OverflowMenuDirective;
+	@Input() open = false;
+
+	@Output() openChange = new EventEmitter<boolean>();
 
-	open = false;
+	@ContentChild(OverflowMenuDirective) overflowMenuDirective: OverflowMenuDirective;
 
 	constructor(protected elementRef: ElementRef, protected i18n: I18n) {}
+
+	handleOpenChange(event: boolean) {
```

Comment:
Not sure where it's coming from but when I open an overflow menu I get the following error:

```
TypeError: Cannot set property 'tabIndex' of undefined
    at OverflowMenuPane.push../src/dialog/overflow-menu/overflow-menu-pane.component.ts.OverflowMenuPane.afterDialogViewInit (overflow-menu-pane.component.ts:128)
    at dialog.component.ts:154
    at ZoneDelegate.push../node_modules/zone.js/dist/zone.js.ZoneDelegate.invokeTask (zone.js:421)
    at Object.onInvokeTask (core.js:17290)
    at ZoneDelegate.push../node_modules/zone.js/dist/zone.js.ZoneDelegate.invokeTask (zone.js:420)
    at Zone.push../node_modules/zone.js/dist/zone.js.Zone.runTask (zone.js:188)
    at push../node_modules/zone.js/dist/zone.js.ZoneTask.invokeTask (zone.js:496)
    at ZoneTask.invoke (zone.js:485)
    at timer (zone.js:2054)
```

And opening a partially visible overflow menu doesn't work, maybe that change wasn't pulled yet?

Reply:
This lead me down a whole rabbit hole of refactoring the dialog service ... but, everything works like 1000% better now, so that's good 🎉